src/components/BookForm.js

```diff
@@ -1,24 +1,44 @@
 import styled from 'styled-components';
+import { useDispatch } from 'react-redux';
+import { v4 as generateID } from 'uuid';
+import { useState } from 'react';
+import { addBook } from '../redux/books/books';
+
+const FormContainer = styled.div`
+  border-top: 1px solid #e8e8e8;
+`;
+
+const AddBook = styled.h2`
+  font-size: 1.25rem;
+  font-weight: bold;
+  letter-spacing: -0.18px;
+  color: #888;
+`;
 
 const BookForm = () => {
-  const FormContainer = styled.div`
-    border-top: 1px solid #e8e8e8;
-  `;
+  const dispatch = useDispatch();
+
+  const [title, setTitle] = useState('');
+  const id = generateID();
 
-  const AddBook = styled.h2`
-    font-size: 1.25rem;
-    font-weight: bold;
-    letter-spacing: -0.18px;
-    color: #888;
-  `;
+  const submitBookToStore = (e) => {
+    e.preventDefault();
+    const newBook = {
+      id,
+      title,
+      author: 'Robert',
+    };
+    dispatch(addBook(newBook));
+    setTitle('');
+  };
 
   return (
     <FormContainer>
       <AddBook>ADD NEW BOOK</AddBook>
-      <form>
-        <input type="text" name="name" placeholder="Book title" />
-        <select>
-          <option disabled selected>Category</option>
+      <form onSubmit={submitBookToStore}>
+        <input type="text" onChange={(e) => setTitle(e.target.value)} value={title} placeholder="Book title" />
+        <select defaultValue="DEFAULT">
+          <option value="DEFAULT" disabled>Category</option>
         </select>
```

Comment:
- [ ] Good job so far 👍. However, instead of having a default author `author: 'Robert'` each time a book is created, and a default `category` option, why don't you add several categories in which users can select any category they like out of the ones you provided or you can remove this category `select` tag and use normal `input` tag for the author just like the input for the title where you allow users to add `author` of the book they added 🤔. Think about this and choose one option and implement it instead of having it as default. 👍